deep_quoridor/src/agents/greedy.py

```diff
@@ -172,18 +182,16 @@ def _log_action(
         # Log the possible next movements
         movement_mask = action_mask[: self.board_size**2]
         for action in np.argwhere(movement_mask == 1).reshape(-1):
-            # self.action_log.action_text(self.action_encoder.index_to_action(int(action)), "")
-            pass
+            self.action_log.action_text(self.action_encoder.index_to_action(int(action)), "")
 
-        my_coords = np.argwhere(observation["board"] == 1)
-        path = my_shortest_path[:]
-        path.insert(0, (int(my_coords[0][0]), int(my_coords[0][1])))
-        # self.action_log.path(path)
+        self.action_log.path(my_shortest_path)
+        self.action_log.path(opponent_shortest_path)
 
```

Comment:
Now that the path includes the coordinates of the pawn we don't need to manually add them!